### DISCUSSION TITLE
It's a thanx note not an issue

### DISCUSSION THREAD

**Discussion by gmareater** [saucecontrol/BrotliIIS#6] [General]
The time and effort you put to complete the documentation was really helpful :)
without it we were unable to run it this fast and easy and you had answer for all the problem we faced :) 

That was great (at the end out manager decided to go with Microsoft release because as the thing you can guess)

but without your documentation it was not possible to get it run.

Keep it up bro, we really appreciate what you've done
Thank you

**Comment by saucecontrol** [comment]
Hey, thanks for the words of encouragement!  I'm glad the docs helped. :)
